snap-wrappers/bin/launch-multipass-gui

```diff
@@ -3,4 +3,14 @@
 . client-common.sh
 link_autostart
 
+if [ ! -e "${XDG_CONFIG_HOME}/xfce4/terminal/terminalrc" ]; then
+  mkdir -p "${XDG_CONFIG_HOME}/xfce4/terminal/"
+  cat <<EOT > "${XDG_CONFIG_HOME}/xfce4/terminal/terminalrc"
+[Configuration]
+Encoding=UTF-8
+MiscMenubarDefault=FALSE
+ShortcutsNoMenukey=TRUE
```

Comment:
Please add this:

```suggestion
ShortcutsNoMenukey=TRUE
FontUseSystem=TRUE
```

Reply:
Why? That looks horrible!

![Screenshot from 2021-08-31 14-21-03](https://user-images.githubusercontent.com/9048879/131509895-0eb175c4-78af-4e7e-933d-ff8a8e0a4bb6.png)